### PR TITLE
Pep8 fixes for rabbitmq modules

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
* PEP8 Fixes
* Code refactoring

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
- lib/ansible/modules/messaging/rabbitmq_binding.py
- lib/ansible/modules/messaging/rabbitmq_exchange.py
- lib/ansible/modules/messaging/rabbitmq_parameter.py
- lib/ansible/modules/messaging/rabbitmq_plugin.py
- lib/ansible/modules/messaging/rabbitmq_policy.py
- lib/ansible/modules/messaging/rabbitmq_queue.py
- lib/ansible/modules/messaging/rabbitmq_user.py
- lib/ansible/modules/messaging/rabbitmq_vhost.py

##### ANSIBLE VERSION
```
2.4 devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
Simpler version - https://github.com/ansible/ansible/pull/24590/files?w=1

**Comment by Akasurde** [comment]
ready_for_review

**Comment by Akasurde** [comment]
ready_for_review

File: lib/ansible/modules/messaging/rabbitmq_binding.py [hunk 1]
```diff
@@ -116,119 +116,113 @@
...
+                })
             )
         elif module.params['state'] == 'absent':
-            r = requests.delete( url, auth = (module.params['login_user'],module.params['login_password']))
+            r = requests.delete(url, auth=(module.params['login_user'], module.params['login_password']))
 
         if r.status_code == 204 or r.status_code == 201:
-            module.exit_json(
-                changed = True,
-                name = module.params['name'],
```
Comment (resmo) [inline lib/ansible/modules/messaging/rabbitmq_binding.py:214]: It seems name param removed from the return, but it should not.

File: lib/ansible/modules/messaging/rabbitmq_exchange.py [hunk 2]
```diff
@@ -118,112 +118,107 @@
...
             module.fail_json(
-                msg = "RabbitMQ RESTAPI doesn't support attribute changes for existing exchanges"
+                msg="RabbitMQ RESTAPI doesn't support attribute changes for existing exchanges"
             )
 
     # Exit if check_mode
     if module.check_mode:
-        module.exit_json(
-            changed= change_required,
-            name = module.params['name'],
```
Comment (resmo) [inline lib/ansible/modules/messaging/rabbitmq_exchange.py]: Same here, what is the reason to remove the name from the return?

File: lib/ansible/modules/messaging/rabbitmq_exchange.py [hunk 3]
```diff
@@ -118,112 +118,107 @@
...
+                })
             )
         elif module.params['state'] == 'absent':
-            r = requests.delete( url, auth = (module.params['login_user'],module.params['login_password']))
+            r = requests.delete(url, auth=(module.params['login_user'], module.params['login_password']))
 
         if r.status_code == 204:
-            module.exit_json(
-                changed = True,
-                name = module.params['name']
```
Comment (resmo) [inline lib/ansible/modules/messaging/rabbitmq_exchange.py]: here as well.

File: lib/ansible/modules/messaging/rabbitmq_exchange.py [hunk 4]
```diff
@@ -118,112 +118,107 @@
...
-                details = r.text
+                msg="Error creating exchange",
+                status=r.status_code,
+                details=r.text
             )
 
     else:
-        module.exit_json(
-            changed = False,
-            name = module.params['name']
```
Comment (resmo) [inline lib/ansible/modules/messaging/rabbitmq_exchange.py]: and here :)

**Review by resmo — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/messaging/rabbitmq_binding.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/messaging/rabbitmq_binding.py:214]: Are you referring [this](https://github.com/ansible/ansible/pull/24590/files/aee05ea2ae7c5e7cdd84e9037c080b39f0f8e66e#diff-d33fa8ead78fe967669bddbe3e490f6fR140)

Comment (resmo) [reply lib/ansible/modules/messaging/rabbitmq_binding.py:214]: Yes, never mind.

**Review by resmo — APPROVED** [review]

**Comment by Akasurde** [comment]
ready_for_review

**Comment by Akasurde** [comment]
ready_for_review

**Review by resmo — APPROVED** [review]
shipit

**Comment by Akasurde** [comment]
@resmo Could you please merge this PR ?

**Comment by resmo** [comment]
shipit

since this is a community module, asking @gundalow to review to get 2 shipits

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Rebase needed

**Comment by Akasurde** [comment]
@gundalow Done with rebase.

**Comment by Akasurde** [comment]
@resmo @gundalow Thanks for review and merge.

**Pull request merged**